_posts/en/newsletters/2018-10-09-newsletter.md

```diff
@@ -0,0 +1,283 @@
+  restrict what outputs a particular coin can be spent to.  For example,
+  you could require that your bitcoins can only be sent to an address
+  that requires giving you 1,000 blocks (1 week) to evaluate the
+  situation and possibly send them back to a whitelisted address before
+  they can be spent to any non-whitelisted address.  (The presenter
+  notes that some developers are opposed to enabling this for
+  fungibility reasons, although alternative approaches may be more
+  acceptable.)
+
+- **Fixing the time warp bug:** a set of miners controlling a majority of
+  the hash rate can currently manipulate Bitcoin's difficulty adjustment
+  algorithm to allow them to consistently create more than one block
+  every ten minutes even without increasing overall hash rate.  There's
+  at least one simple proposal to reduce the amount of manipulation
+  possible without breaking older software or mining equipment.  See the
+  recent [email thread][bitcoin-dev timewarp] on the Bitcoin-Dev mailing
+  list.
+
+- **Explicit fees:** currently fees in Bitcoin are implied by the difference
+  between the value of the aggregated inputs and the aggregated outputs.
+  However, the transaction could alternatively explicitly commit to the
+  fee and allow one of the outputs to be set to the difference between
+  the value of the aggregated inputs and the explicit fee plus all the
+  other outputs.  This could be useful for rewarding Lightning Network
+  watchtowers that send breach remedy transactions on behalf of offline
+  users, or it could be useful for fee bumping group transactions.
+
+However, the presenter concluded this section of the wrap-up by saying,
+"the only people who have comfort with soft-forks are unlikely to
+propose a soft-fork and produce software that would be adopted. People
+are going to fight anything that adds anything, especially considering
+the recent [CVE][CVE-2018-17144]. People are going to be for the next 6
+months significantly more conservative. It's going to be another 6
+months before people are even thinking about it. I don't think we're
+going to get any new soft-forks in the next year."
+
+*(no video), [transcript][tx script], soft forks*
+
+## Footnotes
+
+[^freq-pow-waste]:
+    When a miner creates a new block at the tip of the chain, he can
+    begin working on the next block immediately---but every other miner
+    is still working on an old block until they receive the new block,
+    meaning their proof of work during that brief period of time is
+    wasted (it neither increases network security nor provides the
+    miners with financial compensation).  Miners with more hash rate
+    produce more blocks on average, so they get the head start more
+    often and less of their proof of work is wasted.
+
+    For two perfectly fair miners half a world apart, the network delay
+    can be about one second, meaning a small miner far away from most
```

Comment:
How did you come to the one second delay?

Reply:
Years ago I'd occasionally need to ping servers in western Australia, which is within 500 miles of being the opposite side of the world from where I live, and the RTT was 700ms---so I rounded up to 1,000ms = 1 second.  Now that I write that, I realize that RTT is 2x the minimum relay time, giving me 350ms which is in line with the figures I've found in a quick survey of Google search results suggesting 200ms is roughly the floor for practical packet routing.  I'll update this footnote to use 200ms as a baseline.

Thanks for calling out my shoddy assumption!